Dfe.PrepareTransfers.Web/Services/DocumentGenerators/RisksGenerator.cs

```diff
@@ -0,0 +1,49 @@
+using Dfe.PrepareTransfers.Data.Models.Projects;
+using Dfe.PrepareTransfers.DocumentGeneration;
+using Dfe.PrepareTransfers.DocumentGeneration.Elements;
+using Dfe.PrepareTransfers.Helpers;
+using Dfe.PrepareTransfers.Web.Models.ProjectTemplate;
+using System.Collections.Generic;
+
+namespace Dfe.PrepareTransfers.Web.Services.DocumentGenerators
+{
+    public static class RisksGenerator
+    {
+        public static void AddRisks(DocumentBuilder builder, ProjectTemplateModel projectTemplateModel)
+        {
+            builder.ReplacePlaceholderWithContent("RisksInformation", build =>
+            {
+                build.AddTextHeading("Risks", HeadingLevel.One);
+
+                if (projectTemplateModel.AnyIdentifiedRisks.Equals(true) )
+                {
+                      foreach (var item in projectTemplateModel.ListOfOtherFactors)
+                    {
+
+                        build.AddTable(new List<TextElement[]>
+                        {
+                         new[] { new TextElement { Value = EnumHelpers<TransferBenefits.OtherFactor>.GetDisplayValue(item.Key), Bold = true }, new TextElement { Value = item.Value } },
```

Comment:
I think it's better to not have a new table for each risk, rather a new row for each risk. In summary if you move the foreach down to the text element level.

Reply:
Ah I've just seen Dan G's screenshots and it renders as rows regardless, so I'm happy with that.